boot.sh

```diff
@@ -1,6 +1,10 @@
 #!/bin/sh
 mkdir instance
-FLASK_APP=servicex/app.py flask db upgrade
-#flask db upgrade
-#flask translate compile
+# SQLite doesn't handle migrations, so rely on SQLAlchmy table creation
+if grep "sqlite:////sqlite/app.db" /opt/servicex/app.conf; then
```

Comment:
Should we hardcode `sqlite/app.db`, or match any URI beginning with `sqlite:////`? I see that `sqlite:///:memory:` is used in the unit test suite, although that isn't run via this boot script.